src/pages/rdfClassHierarchyInfo.html

```diff
@@ -10,8 +10,27 @@
 
 <div class="toolbar-holder pull-right mt-1" id="toolbar"
 	ng-class="{ 'pushed-toolbar': showClassInfoPanel }"
-	ng-show="hasClassHierarchy() && !loader">
-
+	ng-show="hasClassHierarchy() && !loader || !isAllGraphsSelected()">
+    <div id="selectGraphDropdown" class="btn-group" role="group" ng-show="graphsInRepo.length > 2">
```

Comment:
I'd have this dropdown implementation as a directive and reuse it here and in the dependencies view above as it's actually the same, but this is out of scope I guess :)